osu.Game/Screens/Backgrounds/BackgroundModeBeatmap.cs

```diff
@@ -30,20 +30,26 @@ public WorkingBeatmap Beatmap
 
                 beatmap = value;
 
-                Background oldBackground = background;
+                Schedule(() =>

```

Comment:
Is my assumption correct that the schedule call is required because `Game` may not be set yet?